src/System.ComponentModel.EventBasedAsync/src/System/ComponentModel/BackgroundWorker.cs

```diff
@@ -8,7 +8,7 @@
 
 namespace System.ComponentModel
 {
-    public class BackgroundWorker : IDisposable
+    public class BackgroundWorker : System.ComponentModel.Component
```

Comment:
Indeed -- there were quite a few places where I had to remove the Dispose method we introduced before this.